src/librustdoc/html/static/js/main.js

```diff
@@ -265,7 +265,7 @@ function preLoadCss(cssUrl) {
                             aria-label="Run search in the documentation"
                             autocomplete="off"
                             spellcheck="false"
-                            placeholder="Type ‘S’ or ‘/’ to search, ‘?’ for more options…"
+                            placeholder="Search API docs"
```

Comment:
I'd prefer to keep it as is until we come up with a solution on how to tell people that this shortcut exists.

Reply:
Already told to people by help button.

Reply:
The issue metioned by @GuillaumeGomez exists and could be improved in later PR, without necessarily blocking this PR.